packages/react-sdk/src/components/BoardBar/pdfv2/pdf.local.ts

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright 2024 Nordeck IT + Consulting GmbH
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { WhiteboardDocumentExport } from '../../../state/export/whiteboardDocumentExport';
+
+const renderPDF = async (exportData: WhiteboardDocumentExport) => {
+  const { pdf, Font } = await import('@react-pdf/renderer');
```

Comment:
We can not use any file level imports for things that interact with react in anyway as this will break HMR on the webworker due to vite bugs.

Reply:
This ensures the bundles are split correctly across the 2 worlds